ksqldb-common/src/main/java/io/confluent/ksql/properties/PropertiesUtil.java

```diff
@@ -77,6 +77,20 @@ public static Properties asProperties(final Map<String, ?> mapProps) {
     return properties;
   }
 
+  /**
+   * Convert object properties values to its string form.
+   *
+   * @param props The map that contains values of different type.
+   * @return A map with all values converted to strings.
+   */
+  public static Map<String, String> toMapStrings(final Map<String, Object> props) {
+    final Map<String, String> stringsProps = new HashMap<>();
+    for (Map.Entry<String, Object> entry : props.entrySet()) {
+      stringsProps.put(entry.getKey(), entry.getValue().toString());
```

Comment:
nit: might be better to use `String.valueOf()` to avoid a possible NPE

Reply:
Thanks. 
Done